common/src/main/java/org/opensearch/ml/common/transport/register/MLRegisterModelResponse.java

```diff
@@ -18,33 +18,46 @@
 @Getter
 public class MLRegisterModelResponse extends ActionResponse implements ToXContentObject {
     public static final String TASK_ID_FIELD = "task_id";
+    public static final String MODEL_ID_FIELD = "model_id";
     public static final String STATUS_FIELD = "status";
 
     private String taskId;
     private String status;
+    private String modelId;
 
     public MLRegisterModelResponse(StreamInput in) throws IOException {
         super(in);
         this.taskId = in.readString();
         this.status = in.readString();
+        this.modelId = in.readOptionalString();
     }
```

Comment:
Feels like a good improvement for remote models, do we really have a taskId which is doing real work for remote models?

Reply:
Curious what kind of validations ?